### PR TITLE
added new category electronics, included in pattern class and written…

### PR DESCRIPTION
**Pull request by mephi007**
Added new category Electronics
Included in pattern class
Written test class

### CODE REVIEW

File: qual_id/categories/electronic.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from ..category import Category
+
+
+class Electronic(Category):
+  def get_values(self):
+    return [
+        'laptop',
+        'Desktop',
```
Comment (gabrielbarker) [inline qual_id/categories/electronic.py:8]:
```suggestion
        'desktop',
```
This should be lowercase

Comment (gabrielbarker) [inline qual_id/categories/electronic.py:8]:
```suggestion
        'desktop',
```
This should be lowercase

File: qual_id/categories/electronic.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        'refrigerator',
+        'mobile',
+        'television',
+        'radio',
+        'trimmer',
+		'headPhones',
```
Comment (gabrielbarker) [inline qual_id/categories/electronic.py:14]:
```suggestion
	'headphones',
```
No need for camelcase

File: qual_id/categories/electronic.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		'fan',
```
Comment (gabrielbarker) [inline qual_id/categories/electronic.py:15]: please fix the alignment of these elements

File: test/categories/test_electronic.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.electronic import Electronic
+import random
+
+
+class TestElectronic(unittest.TestCase):
```
Comment (gabrielbarker) [inline test/categories/test_electronic.py:6]: If you make sure that you have the latest changes from the original repo, the way to write unit tests has been simplified. See any other test class for reference

**Review by gabrielbarker — CHANGES_REQUESTED** [review]

**Comment by gabrielbarker** [comment]
Hey @mephi007 
I've put some comments on, just some small tweaks, thanks a lot for the new category!
🙂

**Comment by mephi007** [comment]
hey @gabrielbarker, I am not able understand, how build is failing?

File: qual_id/pattern.py [hunk 5]
```diff
@@ -19,6 +20,8 @@ class Pattern:
       "game":Game(),
       "geography":Geography(),
       "instrument": Instrument()
```
Comment (gabrielbarker) [inline qual_id/pattern.py:22]:
```suggestion
      "instrument": Instrument(),
```
You missed a comma @mephi007 
Easy to do, especially in python 🙂

**Review by gabrielbarker — APPROVED** [review]
Thanks for fixing those issues!

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
Once the comma is added this should pass and be ready to go

**Review by gabrielbarker — APPROVED** [review]
Looks good!

**Review by gabrielbarker — APPROVED** [review]

**Comment by gabrielbarker** [comment]
Thanks again @mephi007 , nice work 🙂

**Comment by mephi007** [comment]
Thank you so much @gabrielbarker

**Pull request merged**